stats/views.go

```diff
@@ -24,11 +23,12 @@ var (
 	// CheckRuntime tracks CPU runtime stats.
 	CheckRuntime = view.View{
 		Name:        "CheckRuntime",
-		Description: "CPU runtime stats per repo per check",
+		Description: "CPU runtime stats per check",
 		Measure:     CPURuntimeInSec,
-		TagKeys:     []tag.Key{Repo, CheckName},
```

Comment:
do we want a separate view with just the "Repo" as the tag? Maybe some repos result in more with CPU/HTTP request usage and we want to know about them?

Reply:
Good point. I've left a TODO here for this. If this PR does help with the logs, then can add/enable per-repo CPU runtime stats.